diagram/yang-standalone/package.json

```diff
@@ -5,15 +5,17 @@
   "license": "Apache-2.0",
   "dependencies": {
     "reconnecting-websocket": "^3.0.7",
-    "sprotty": "0.2.1",
-    "yang-sprotty": "0.1.1",
+    "sprotty": "next",
+    "yang-sprotty": "next",
+    "sprotty-theia": "next",
```

Comment:
Why do you need sprotty-theia?

Reply:
Without this, webpack complains about the missing dependency to `sprotty-theia/css/theia-sprotty.css`, which is referenced in [`yang-sprotty/di.config.ts` line 23](https://github.com/theia-ide/yang-sprotty/blob/fb3f0364a9ef378947003dcce4d5fd9fa1807b11/src/di.config.ts#L23).

Reply:
Is that CSS import necessary for yang-eclipse? If not, it could be moved to [yangster](https://github.com/theia-ide/yangster/blob/master/theia-yang-extension/src/frontend/yangdiagram/di.config.ts).